sample_dataset/metabase/sample_dataset/generate.clj

```diff
@@ -60,18 +61,27 @@
      :latitude   (random-latitude)
      :longitude  (random-longitude)
      :source     (rand-nth ["Google" "Twitter" "Facebook" "Organic" "Affiliate"])
-     :created_at (random-date-between (u/relative-date :year -1) (u/relative-date :year 1))}))
+     :created_at (random-date-between (u/relative-date :year -2) (u/relative-date :year 1))}))
 
 ;;; ## PRODUCTS
 
 (defn- random-company-name []
   (first (company/names)))
 
+(defn- rejection-sample
```

Comment:
We don't enforce docstrings for private functions like this, but there's not rule against having them anyway. TBH I'm not a stats expert so the term `rejection-sample` is meaningless to me. It would be great if we even had a short blurb that would help someone playing around in this namespace understand what exactly is going on without having to go ask anyone else

Reply:
`random-price` below this is clear enough but for things that aren't immediately clear to other people who have to work on the code I would prefer we try to at least make the code somewhat documented, private function or no. 

I think fixing the frontend bugs I fixed the last couple weeks took me 3-4x as long as it needed to because it took me so long to understand what the code was supposed to be doing. I think the viz lib averaged maybe one comment per 400 lines of code.

Spending a couple extra minutes writing explanations of what everything that's not immediately obvious does and how it works with without a doubt save us time in the future when we go back to tweak it

Reply:
+1

Reply:
Sorry, this was already fixed